flatten-array.json

```diff
@@ -0,0 +1,19 @@
+{
+   "cases": [
+      {
+         "description": "flattens array with null and integers present",
+         "input": [0, 2, [[2, 3], 8, 100, null, [[null]]], -2],
+         "expected": [0, 2, 2, 3, 8, 100, -2]
+      },
+      {
+         "description": "flattens array with just integers present",
```

Comment:
This seems like a simpler example than the previous one (no nulls vs has nulls). I think it should go before it.